snowplow-tracker/src/main/java/com/snowplowanalytics/snowplow/tracker/Tracker.java

```diff
@@ -568,6 +605,40 @@ public void run() {
 
     // --- Helpers
 
+    private void addServiceEventPayload(Payload payload, List<SelfDescribingJson> contexts, String eventId) {
```

Comment:
Is a ServiceEvent an event from within the tracker that doesn't include any Subject information?
A method comment like addEventPayload would be useful to desribe how this differs from addEventPayload would be great.

Reply:
Good idea!